migrations/initdb-banners.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable new-cap */
+
+'use strict';
+
+const owner = process.env.DATASTORE_SEQUELIZE_OWNER || 'postgres';
+const schema = process.env.DATASTORE_SEQUELIZE_SCHEMA || 'public';
+const lockTimeout = process.env.DATASTORE_SEQUELIZE_LOCKTIMEOUT || '2s';
+const prefix = process.env.DATASTORE_SEQUELIZE_PREFIX || '';
+const table = `${prefix}banners`;
+
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        await queryInterface.sequelize.transaction(async (transaction) => {
+            await queryInterface.sequelize.query(
+                `SET lock_timeout TO '${lockTimeout}';`, { transaction });
```

Comment:
`SET lock_timeout ...` seems to be postgres-specific.
Should it be set via `dialectOptions` in Sequelize option?

Reply:
yes, this I missed, will remove this; better way is to control via DB. Sequelize dialectOptions doesn't support query level timeouts.